cmd/ethrex/bench/build_block_benchmark.rs

```diff
@@ -182,44 +180,29 @@ async fn fill_mempool(b: &Blockchain, accounts: Vec<SecretKey>) {
     }
 }
 
-pub async fn bench_payload(input: &(&mut Blockchain, Block, &Store)) -> (Duration, u64) {
```

Comment:
looking at the name of this function, I have no idea what it does. Maybe some doc would be useful.

Reply:
Would you prefer `bench_payload_build`?